packages/sonarwhal/src/lib/cli.ts

```diff
@@ -67,7 +67,16 @@ const notifyIfNeeded = () => {
 
 /** Executes the CLI based on an array of arguments that is passed in. */
 export const execute = async (args: string | Array<string> | Object): Promise<number> => {
-    const currentOptions: CLIOptions = options.parse(args);
+    let currentOptions: CLIOptions;
+
+    try {
+        currentOptions = options.parse(args);
+    } catch (e) {
+        logger.error(e.message);
```

Comment:
I think we should show the help message.

Reply:
Does `e.message` contain the help message?

If not, I think you should show it.

Reply:
@sarvaje Yes `e.message` contains the help message.

Reply:
ok, perfect!